packages/api/src/graphql.js

```diff
@@ -1,111 +1,113 @@
 "use strict";
 
+const { defaultFieldResolver } = require("graphql");
```

Comment:
The default resolver lets us not have to specify [the simple property fetching resolvers](#diff-fd5891aeaa0b07b6dec7c1130af79cf6L95-L100)